README.md

```diff
@@ -130,6 +130,8 @@ This file must be located in the project root and declares the project settings.
 | assets                    |                             | static asset files, that should be copied to the styleguide directory, head for "Assets" for further information
 | content                   |                             | list of page configurations, take a look at "Content"
 | componentDocs             | "."                         | path of component doc files, relative to the given component.yml. "doc" e.g says "all documents have to be placed in a sub-directory doc adjacent to the component.yml"
+| assetHost                 |                             | url origin for external assets
+| themes                    |                             | list of style themes, of that each defines a different themed version of the components
```

Comment:
There is already a theme property introduced, which relates to the choice of theme-flatwhite or taking something else. While there are no other themes yet, i decided to not document it yet. 

But i think "themes" is a little bit to generic. Maybe we should call it something like "componentThemes" in the public API, to highlight the differences?